app/models/file_depot_ftp.rb

```diff
@@ -93,12 +93,15 @@ def file_exists?(file_or_directory)
   private
 
   def create_directory_structure(directory_path)
-    Pathname.new(directory_path).descend do |path|
```

Comment:
Looks good to me.  I'd like an examples for iis and anonymous ftp that @jcarter12 before merge though.  We've changed things for other ftps in the past and I feel like this might cause a regression without confirmation.